roles/openshift_logging/README.md

```diff
@@ -12,7 +10,156 @@ generation for Elasticsearch (it uses JKS) as well as openssl to sign certificat
+    # extra namespaces to create for mux clients - users will need to set this
+    namespaces: []
+
+  es:
+    install: False
+    master_url: "https://kubernetes.default.svc.{{ openshift.common.dns_domain }}"
+    master_public_url: "{{ 'https://' + openshift_master_cluster_public_hostname | default(openshift.common.public_hostname, true) + ':' ~ openshift_master_api_port }}"
+    nodeselector: null
+    labels: {}
+    purge_logging: False
+    image_pull_secret: ""
+
+    curator:
+      default_days: 30
+      run_hour: 3
+      run_minute: 30
+      run_timezone: UTC
+      timeout: 300
+      script_log_level: INFO
+      log_error: ERROR
+      cpu_limit: null
+      cpu_request: 100m
+      memory_limit: 256Mi
+      node_selector: {}
+
+    kibana:
+      hostname: "{{ 'kibana.' ~ openshift_master_default_subdomain }}"
+      cpu_limit: null
+      cpu_request: 100m
+      memory_limit: 736Mi
+      replica_count: 1
+      edge_term_policy: Redirect
+      nodeselector: {}
+      cert: ""
+      key: ""
+      ca: ""
+      proxy:
+        debug: false
+        cpu_limit: null
+        cpu_request: 100m
+        memory_limit: 256Mi
+
+    elasticsearch:
+      host: logging-es
+      port: 9200
+      ca: /etc/fluent/keys/ca
+      client_cert: /etc/fluent/keys/cert
+      client_key: /etc/fluent/keys/key
+      cluster_size: 1
+      cpu_limit: null
+      cpu_request: "1"
+      # the logging appenders for the root loggers to write ES logs. Valid values: 'file', 'console'
+      log_appenders: ['file']
+      memory_limit: "8Gi"
+      pv_selector: "{{ openshift_logging_storage_labels | default('') }}"
+      pvc_dynamic: "{{ openshift_logging_elasticsearch_pvc_dynamic | default(False) }}"
+      pvc_size: ''
+      pvc_prefix: "{{ openshift_logging_elasticsearch_pvc_prefix | default('logging-es') }}"
+      recover_after_time: 5m
+      storage_group: "65534"
+      nodeselector: {}
+      # openshift_logging_es_config is a hash to be merged into the defaults for the elasticsearch.yaml
+      config: {}
+
+      # for exposing es to external (outside of the cluster) clients
+      allow_external: False
+      hostname: "{{ 'es.' ~ openshift_master_default_subdomain }}"
+
+      #The absolute path on the control node to the cert file to use
+      #for the public facing es certs
+      cert: ""
+
+      #The absolute path on the control node to the key file to use
+      #for the public facing es certs
+      key: ""
+
+      #The absolute path on the control node to the CA file to use
+      #for the public facing es certs
+      ca_ext: ""
+
+      # allow cluster-admin or cluster-reader to view operations index
+      # This is really only applicable for your infra cluster
+      allow_cluster_reader: False
+```
+
+To deploy two clusters, one for apps and one for infra logs, with defaults as was previously accomplished using `openshift_logging_use_ops`,
+you would specify multiple cluster names like this:
```

Comment:
Are we saying that a single run of ansible would deploy both clusters are are we trying to say it would deploy the 'app' cluster and point fluent to the ops cluster?  I can image the case we worked with ops where cluster A I want the full stack but cluster B I only want elastic and Kibana.  I would think I would need to run ansible twice.  Is that represented here?

Reply:
So far with this structure, I think that we can do this running ansible once, yes. 

As of the current commits this isn't shown but the plan is each cluster definition will also have its own namespace and we can configure fluentd to point to any app or infra cluster (as we can do currently).